hack/common.sh

```diff
@@ -41,24 +41,29 @@ readonly OS_SCRATCH_IMAGE_COMPILE_TARGETS=(
   examples/deployment
 )
 readonly OS_IMAGE_COMPILE_BINARIES=("${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]##*/}" "${OS_IMAGE_COMPILE_TARGETS[@]##*/}")
-
 readonly OS_CROSS_COMPILE_PLATFORMS=(
   linux/amd64
   darwin/amd64
   windows/amd64
   linux/386
 )
-readonly OS_CROSS_COMPILE_TARGETS=(
+
+if [ "_$@" = "_test/e2e/e2e.test" ]; then
```

Comment:
that said, it is super weird to use the same var ($OS_CROSS_COMPILE_TARGETS), but then special case it in the build loop (https://github.com/openshift/origin/pull/7049/files#diff-f4e81489ff94a80b6c06c769fac10d80R234). If we can't use the build-loop as-is, we shouldn't be putting it in this var. We already do different things to build tests (https://github.com/openshift/origin/pull/7049/files#diff-f4e81489ff94a80b6c06c769fac10d80R223). I would expect to use that, or have another loop if the e2e binary needs other test flags set